libdnf/base/goal.cpp

```diff
@@ -406,13 +405,13 @@ void Goal::Impl::add_upgrades_to_goal() {
             // TODO(jmracek) Replace obsoletes_query.get_package_set(); by more effective approach
```

Comment:
Please could you remove `// TODO(jmracek) Replace obsoletes_query.get_package_set(); by more effective approach`. It is already resolved.

Reply:
OK. I will remove.